### ISSUE TITLE
False positive for 930130 with .pac

### ISSUE THREAD

**Issue by S0obi** [coreruleset/coreruleset#4452]
### Description

This change https://github.com/coreruleset/coreruleset/pull/4299 seems to have introduce a new entry in restricted-files.data‎ : `.pac`

### How to reproduce the misbehavior (-> curl call)

No sure if sandbox is already 4.23.0 ?

But a request like this will trigger 930130 : `https://example.com/assets/framework.pack.js` because it contains .pac.

### Logs

```raw
ModSecurity: Warning. Matched "Operator `PmFromFile' with parameter `restricted-files.data' against variable `REQUEST_FILENAME' (Value: `/assets/framework.pack.js' ) [file "/etc/nginx/modsec/coreruleset-4.23.0/rules/REQUEST-930-APPLICATION-ATTACK-LFI.conf"] [line "126"] [id "930130"] [rev ""] [msg "Restricted File Access Attempt"] [data "Matched Data: .pac found within REQUEST_FILENAME: /assets/framework.pack.js"] [severity "2"] [ver "OWASP_CRS/4.23.0"] [maturity "0"] [accuracy "0"] [tag "application-multi"] [tag "language-multi"] [tag "platform-multi"] [tag "attack-lfi"] [tag "paranoia-level/1"] [tag "OWASP_CRS"] [tag "OWASP_CRS/ATTACK-LFI"] [tag "capec/1000/255/153/126"] [hostname "wa-sqa-tt029.ocs-support.com"] [uri "/assets/framework.pack.js"] [unique_id "177074068142.707258"] [ref "o48,4v4,56t:utf8toUnicode,t:urlDecodeUni,t:normalizePathWin"]
```

### Your Environment

* CRS version : v4.23.0
* Paranoia level setting : PL1
* ModSecurity version : latest
* Web Server and version : nginx latest
* Operating System and version: Ubuntu 22.04

### Confirmation

[X] I have removed any personal data (email addresses, IP addresses,
    passwords, domain names) from any logs posted.

**Comment by fzipi** [comment]
Fixed in #4451
